src/core/providers/ogr/qgsogrprovider.cpp

```diff
@@ -1601,7 +1603,9 @@ bool QgsOgrProvider::addFeaturePrivate( QgsFeature &f, Flags flags, QgsFeatureId
           QString stringValue;
 
           if ( OGR_Fld_GetSubType( fldDef ) == OFSTJSON )
-            stringValue = jsonStringValue( attrVal );
```

Comment:
Should QgsOgrProvider::jsonStringValue be removed now? It looks unused after these changes, and if it's buggy then we don't want someone accidentally using it in future...

Reply:
I don't think it's buggy, it's just don't needed. But I agree it can be removed if not used.

Reply:
I checked and it's still used, when the subtype is not JSON. I'll leave it.